src/App.js

```diff
@@ -1,25 +1,44 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import { ListItem } from "./components/ListItem";
+import "./components/style.css";
+
+export default function App() {
+  const [text, setText] = useState("");
+  const [toDoList, updateToDoList] = useState([]);
+  const addNote = () => {
+    updateToDoList([...toDoList, { id: Math.random(), description: text }]);
+    // toDoList.push({ description: text });
+    // updateToDoList(toDoList);
+    setText("");
+  };
+
+  function deleteToDo(id) {
+    // filter out the item with the id
+    const updatedToDoList = toDoList.filter((item) => item.id !== id);
+    updateToDoList(updatedToDoList);
+  }
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app-background">
+      <p className="heading-text">React-to-do-list</p>
+      <div className="task-container">
+        <div>
+          <input
+            className="text-input"
+            placeholder="type here..."
+            value={text}
+            onChange={(event) => setText(event.target.value)}
```

Comment:
```suggestion
            onChange={(event) => {
              setText(event.target.value)}
            }
```

No implicit return in void functions.